include/quda.h

```diff
@@ -1531,6 +1531,22 @@ extern "C" {
   void computeCloverForceQuda(void *mom, double dt, void **x, void **p, double *coeff, double kappa2, double ck,
 			      int nvector, double multiplicity, void *gauge,
 			      QudaGaugeParam *gauge_param, QudaInvertParam *inv_param);
+  
+  /** this is a minimal test example of a function which computes the force
```

Comment:
I'm guessing this doxygen needs an update, since this isn't a minimal test example (anymore?)

Reply:
014913e504d47481b3c15b71084b2986d3339831